src/main/java/gyro/aws/cloudfront/CloudFrontResource.java

```diff
@@ -698,4 +729,15 @@ private DistributionConfig distributionConfig() {
 
         return builder.build();
     }
+
+    private void applyMonitoringSubscription(CloudFrontClient client, boolean isUpdate) {
+        if (getMonitoringSubscription()) {
+            client.createMonitoringSubscription(r -> r
```

Comment:
Add distributionId()
```suggestion
            client.createMonitoringSubscription(r -> r.distributionId(getId())
```